src/_posts/languages/php/2000-01-01-start.md

```diff
@@ -64,57 +68,50 @@ The default PHP version on both `scalingo-20` and `scalingo-22` is the latest
 ```
 
 {% note %}
-  You should not specify a precise version such as `8.2.3` or you would miss
-  important updates. Instead, you should specify `~8.2.3` to install a version
-  `>=8.2.3 and <8.3.0` or specify `~8.2` to install a version `>=8.2 and
-  <9.0.0`.
+You should not specify a precise version such as `8.2.3` or you would miss
+important updates. Instead, you should specify `~8.2.3` to install a version
+`>=8.2.3 and <8.3.0` or specify `~8.2` to install a version `>=8.2 and <9.0.0`.
 
-  Further details about version constraints can be found in the
-  [Composer documentation](https://getcomposer.org/doc/articles/versions.md#writing-version-constraints)
+Further details about version constraints can be found in the
+[Composer documentation](https://getcomposer.org/doc/articles/versions.md#writing-version-constraints)
 {% endnote %}
 
+
 ## PHP Extensions and Dependencies
 
 See our dedicated documentation pages:
 
 - [Working with PHP Dependencies]({% post_url languages/php/2000-01-01-dependencies %})
 - [Working with PHP Extensions]({% post_url languages/php/2000-01-01-extensions %})
 
-## Officially Supported Frameworks
 
-Scalingo supports out-of-the-box many well-known frameworks. When using such
-frameworks, you have nothing special to configure in your `composer.json`,
-`git push` your code and everything will work!
+## Supported Frameworks
 
-List of the frameworks:
+Here is a non-exhaustive list of frameworks known to work on Scalingo:
 
 * [Symfony]({% post_url languages/php/2000-01-01-symfony %})
 * [Laravel]({% post_url languages/php/2000-01-01-laravel %})
-* [Slim](https://sample-php-slim.scalingo.io/)
-* CakePHP
-* Lumen
-* Zend Framework 2
-* Magento
```

Comment:
question: Magento does not work and/or is deprecated?
(It is not listed as is in the issue)

Reply:
I haven't been able to test it, Adobe's repo answer with a 404.
So for now I thought it would be safer to not include it. I'll check regularly and update accordingly.

Reply:
Interesting, I can access their GitHub's one:
https://github.com/magento/magento2

Maybe you need a key or something to be able to download it :thinking:

Reply:
https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/overview.html